install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -70,6 +58,10 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Search:      "open-vsx.org",
 						Replacement: openVSXProxyUrl,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
+					}, {
+						Search:      "https://raw.githubusercontent.com/gitpod-io/openvsx.org.json/main/marketplace.json",
+						Replacement: fmt.Sprintf("ide.%s/code/marketplace.json", ctx.Config.Domain),
+						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
```

Comment:
This is not correct, 

Option1: remove https protocol in `Search`

Option2: add https  protocol in `Replacement`

See also request in preview env, the URL should be `https://ide.ft-add-ext649a97d67d.preview.gitpod-dev.com/code/marketplace.json`
![image](https://user-images.githubusercontent.com/20944364/180153748-4ea382b0-d72f-4240-8e0d-98f847cf3558.png)

Reply:
Thank you for catching this, silly me 😆.